draft-linus-trans-gossip-ct.md

```diff
@@ -216,18 +218,18 @@ before storing them:
   discarded
 
   1. if the leaf cert is not for a domain that the server is
-  authoritative for, the SCT MUST be discarded
+  authoritative for, the SCT MUST be discarded [why? perhaps the client doesn't care about privacy? e.g. it is a crawler]
```

Comment:
Scalability concerns should not be normative. If you want to talk about them, add some implementation notes.